services/net/image/ImageAction.cs

```diff
@@ -73,7 +73,7 @@ public ImageAction(IKafkaMessenger producer, IApiService api, IOptions<ImageOpti
         this.Logger.LogDebug("Performing ingestion service action for data source '{name}'", manager.Ingest.Name);
 
         // Extract the ingest configuration settings.
-        var inputFileCode = String.IsNullOrWhiteSpace(manager.Ingest.GetConfigurationValue("code")) ? manager.Ingest.Source?.Code : manager.Ingest.GetConfigurationValue("code");
+        var inputFileCode = String.IsNullOrWhiteSpace(manager.Ingest.SourceConnection?.GetConfigurationValue("code")) ? manager.Ingest.Source?.Code : manager.Ingest.SourceConnection?.GetConfigurationValue("code");
```

Comment:
I don't think this is correct.  The code would be specific to an `Ingest.Configuration`, not the `SourceConnection.Configuration`.  Or at least it makes more sense to me that the connection is more generic about where files are, and the ingest is all about how to ingest those files.

Reply:
Your concern likely relates to Alessia's interpretation of what `code` means in the connection configuration. This is what she sent me as example content for the image service's connection.configuration field:

{"path":"/dsk98/binaryroot/2022/09/27","username":"ckayfish","hostname":"scharnhorst.tno.gov.bc.ca","keyFileName":"id_rsa","code":"sv-GLB"}

In this case, she used `code` as the file name pattern to filter on. Does this provide clarity?

Reply:
Then I believe my understanding is correct.  The `code` should exist in the `Ingest.Configuration` as it is specific to the ingest, not the connection.

Reply:
Reverted it to the way it worked before.